Dockerfile

```diff
@@ -27,6 +27,10 @@ ENTRYPOINT dotnet watch --project ./FFCDemoPaymentService run --urls "http://*:$
 FROM defradigital/dotnetcore:${PARENT_VERSION} AS production
 ARG PARENT_VERSION
 LABEL uk.gov.defra.ffc.parent-image=defradigital/dotnetcore:${PARENT_VERSION}
+
+USER root
+RUN apk --no-cache add postgresql-client
```

Comment:
What role does this package perform?  Is it something we need in every postgres container or is it specific to .NET Core?

Reply:
This was just for testing to get `psql` installed on the pod. Now removed.